outlines/publish-packages.md

```diff
@@ -0,0 +1,27 @@
+# Publishing a Package
+
+1. Pros/cons of publishing a package
+  1. Considering what's involved in publishing a package.
+  2. Do you think you'll be able to maintain it properly?
+  3. Is the code complete enough to be useful, or does it need work to be made generic?
+2. How to decide what's in a package / more than one
+  1. Is there any core functionality that could be reused elsewhere? (e.g. route matching)
+  2. Is there pure JS code that could be published to NPM?
+  3. Principle of a single symbol per package could help (see structure chapter).
+3. How to write great documentation.
+  1. README vs longer documentation/guide
```

Comment:
OTOH I honestly think it's better to have a shorter README for people to get up to speed quickly and then more complex deep-dive docs somewhere else, atmosphere issue aside. That's what we did for iron router and I think it works well.